pkg/crc/machine/machine.go

```diff
@@ -136,6 +155,11 @@ func Start(startConfig StartConfig) (StartResult, error) {
 			return *result, errors.Newf("Error getting bundle metadata: %v", err)
 		}
 
+		if err := checkDiskImageSize(crcBundleMetadata); err != nil {
```

Comment:
only this should be here...

... as this is part of the metadata check function, it makes sense to move the check code into the metadata.go file.

Reply:
@gbraad Ack